packages/arcgis-rest-places/test/findPlacesNearPoint.test.ts

```diff
@@ -74,4 +74,22 @@ describe("findPlacesNearPoint()", () => {
     expect(nextPage.nextPage).toBeUndefined();
     expect(url).toContain("token=MOCK_KEY");
   });
+
+  it("verify endpoint", async () => {
+    fetchMock.mock("*", placeNearPointMockNoMoreResults);
+
+    const response = await findPlacesNearPoint({
```

Comment:
In these tests files, since we're not using the `response` variable, I think you can leave that off, like this:
```suggestion
    await findPlacesNearPoint({
```